libs/sdk/src/Constants.ts

```diff
@@ -10,4 +10,4 @@ export const AUTO_COLLECT_ADVERT_ID = false;
 export const TRACK_LIFECYCLE_EVENTS = true;
 export const RECORD_SCREEN_VIEWS = false;
 export const LOG_LEVEL = RUDDER_LOG_LEVEL.ERROR;
-export const SDK_VERSION = '1.6.3';
```

Comment:
Can't we bump this through CI?

Reply:
This is yet to be implemented in React-Native SDK. BTW, I've already added it as a part of future improvement.

Reply:
you can use the version form the package json see how JS sdk does it via the build script in pckage.json